client/src/GovConsentConfig.ts

```diff
@@ -1,21 +1,37 @@
 import { findByKey, parseUrl } from './utils'
 
+export const COOKIE_DAYS = 365
+
 export class GovConsentConfig {
-  uidKey = 'gov_singleconsent_uid'
+  static UID_KEY = 'gov_singleconsent_uid'
+  static CONSENTS_COOKIE_NAME = 'cookies_policy'
+  static PREFERENCES_SET_COOKIE_NAME = 'cookies_preferences_set'
+  static COOKIE_LIFETIME = COOKIE_DAYS * 24 * 60 * 60
   uidFromCookie: string
   uidFromUrl: string
+  apiUrl: string
+
+  constructor(apiUrl?: string) {
+    this.uidFromCookie = findByKey(
+      GovConsentConfig.UID_KEY,
+      document.cookie.split(';')
+    )
+    this.uidFromUrl = findByKey(
+      GovConsentConfig.UID_KEY,
+      parseUrl(location.href).params
+    )
 
-  constructor() {
-    this.uidFromCookie = findByKey(this.uidKey, document.cookie.split(';'))
-    this.uidFromUrl = findByKey(this.uidKey, parseUrl(location.href).params)
+    this.apiUrl = apiUrl || this.getApiUrlFromHtml()
   }
 
-  getApiUrl() {
+  getApiUrlFromHtml() {
```

Comment:
suggestion: I think we can remove this function now that the URL is passed in to the constructor. This was just for testing purposes anyway.